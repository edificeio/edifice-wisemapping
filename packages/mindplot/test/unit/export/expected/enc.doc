<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Word.Document"?><w:wordDocument xmlns:sl="http://schemas.microsoft.com/schemaLibrary/2003/core" xmlns:w="http://schemas.microsoft.com/office/word/2003/wordml" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882" xmlns:wx="http://schemas.microsoft.com/office/word/2003/auxHint" xmlns:aml="http://schemas.microsoft.com/aml/2001/core" xmlns:o="urn:schemas-microsoft-com:office:office">
<o:DocumentProperties>
<o:Title>Artigos GF comentários interessantes</o:Title>
</o:DocumentProperties>
<w:styles>
<w:versionOfBuiltInStylenames w:val="4"/>
<w:latentStyles w:defLockedState="off" w:latentStyleCount="156"/>
<w:style w:type="paragraph" w:default="on" w:styleId="Normal">
<w:name w:val="Normal"/>
<w:rsid w:val="00831C9D"/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:sz w:val="24"/>
<w:sz-cs w:val="24"/>
<w:lang w:val="EN-US" w:fareast="EN-US" w:bidi="AR-SA"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading1">
<w:name w:val="heading 1"/>
<wx:uiName wx:val="Heading 1"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading1"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="32"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading2">
<w:name w:val="heading 2"/>
<wx:uiName wx:val="Heading 2"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading2"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="1"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading3">
<w:name w:val="heading 3"/>
<wx:uiName wx:val="Heading 3"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading3"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="2"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading4">
<w:name w:val="heading 4"/>
<wx:uiName wx:val="Heading 4"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading4"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="3"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading5">
<w:name w:val="heading 5"/>
<wx:uiName wx:val="Heading 5"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading5"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="4"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading6">
<w:name w:val="heading 6"/>
<wx:uiName wx:val="Heading 6"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading6"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="5"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading7">
<w:name w:val="heading 7"/>
<wx:uiName wx:val="Heading 7"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading7"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="6"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading8">
<w:name w:val="heading 8"/>
<wx:uiName wx:val="Heading 8"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading8"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="7"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:i/>
<w:i-cs/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading9">
<w:name w:val="heading 9"/>
<wx:uiName wx:val="Heading 9"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading9"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="8"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="character" w:default="on" w:styleId="DefaultParagraphFont">
<w:name w:val="Default Paragraph Font"/>
<w:semiHidden/>
</w:style>
<w:style w:type="table" w:default="on" w:styleId="TableNormal">
<w:name w:val="Normal Table"/>
<wx:uiName wx:val="Table Normal"/>
<w:semiHidden/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
<w:tblPr>
<w:tblInd w:w="0" w:type="dxa"/>
<w:tblCellMar>
<w:top w:w="0" w:type="dxa"/>
<w:left w:w="108" w:type="dxa"/>
<w:bottom w:w="0" w:type="dxa"/>
<w:right w:w="108" w:type="dxa"/>
</w:tblCellMar>
</w:tblPr>
</w:style>
<w:style w:type="list" w:default="on" w:styleId="NoList">
<w:name w:val="No List"/>
<w:semiHidden/>
</w:style>
<w:style w:type="paragraph" w:styleId="Title">
<w:name w:val="Title"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Title"/>
<w:spacing w:before="240" w:after="60"/>
<w:jc w:val="center"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="28"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="BodyText">
<w:name w:val="Body Text"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="BodyText"/>
<w:spacing w:after="120"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
</w:styles>
<w:body>
<wx:sect>
    <wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Title"/>
</w:pPr>
<w:r>
<w:t>Artigos GF comentários interessantes</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Poorter 1999. Functional Ecology. 13:396-410</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Espécies pioneiras crescem mais rápido do que as não pioneiras</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Tolerância a sombra está relacionada com persistência e não com crescimento</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Chazdon 2010. Biotropica. 42(1): 31–40</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Falar no artigo que esse trabalho fala que é inadequada a divisão entre pioneira e não pioneira devido a grande variação que há entre elas. Além de terem descoberto que durante a ontogenia a resposta a luminosidade muda dentro de uma mesma espécie. Porém recomendar que essa classificação continue sendo usada em curto prazo enquanto não há informações confiáveis suficiente para esta simples classificação. Outras classificações como esta do artigo são bem vinda, contanto que tenham dados confiáveis. Porém dados estáticos já são difíceis de se obter, dados temporais, como taxa de crescimento em diâmetro ou altura, são mais difíceis ainda. Falar que vários tipos de classificações podem ser utilizadas e quanto mais detalhe melhor, porém os dados é que são mais limitantes. Se focarmos em dados de germinação e crescimento limitantes, como sugerem sainete e whitmore, da uma idéia maismrápida e a curto prazo da classificação destas espécies. Depois com o tempo conseguiremos construir classificações mais detalhadas e com mais dados confiáveis.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Here, we develop a new approach that links functional attributes of tree species with studies of forest recovery and regional land-use transitions (Chazdon et al. 2007). Grouping species according to their functional attributes or demographic rates provides insight into both applied and theoretical questions, such as selecting species for reforestation programs, assessing ecosystem services, and understanding community assembly processes in tropical forests (Diaz et al. 2007, Kraft et al. 2008).</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Since we have data on leaf and wood functional traits for only a subset of the species in our study sites, we based our functional type classification on information for a large number of tree species obtained through vegetation monitoring studies.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Our approach avoided preconceived notions of successional behavior or shade tolerance of tree species by developing an objective and independent classification of functional types based on vegetation monitoring data from permanent sample plots in mature and secondary forests of northeastern Costa Rica (Finegan et al. 1999, Chazdon et al. 2007).We apply an independent, prior classification of 293 tree species from our study region into five functional types, based on two species attributes: canopy strata and diameter growth rates for individuals Z10 cm dbh (Finegan et al. 1999, Salgado- Negret 2007).</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Our results demonstrate strong linkages between functional types defined by adult height and growth rates of large trees and colonization groups based on the timing of seedling, sapling, and tree recruitment in secondary forests.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>These results allow us to move beyond earlier conceptual frameworks of tropical forest secondary succession developed by Finegan (1996) and Chazdon (2008) based on subjective groupings, such as pioneers and shade-tolerant species (Swaine &amp; Whitmore 1988).</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Reproductive traits, such as dispersal mode, pollination mode, and sexual system, were ultimately not useful in delimiting tree functional types for the tree species examined here (Salgado-Negret 2007). Thus, although reproductive traits do vary quantitatively in abundance between secondary and mature forests in our landscape (Chazdon et al. 2003), they do not seem to be important drivers of successional dynamics of trees Z10 cm dbh. For seedlings, however, dispersal mode and seed size are likely to play an important role in community dynamics during succession (Dalling&amp;Hubbell 2002).</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Our classification of colonization groups defies the traditional dichotomy between ‘late successional’ shade-tolerant and ‘early successional’ pioneer species. Many tree species, classified here as regenerating pioneers on the basis of their population structure in secondary forests, are common in both young secondary forest and mature forests in this region (Guariguata et al. 1997), and many are important timber species (Vilchez et al. 2008). These generalists are by far the most abundant species of seedlings and saplings, conferring a high degree of resilience in the wet tropical forests of NE Costa Rica (Norden et al. 2009, Letcher &amp; Chazdon 2009). The high abundance of regenerating pioneers in seedling and sapling size classes clearly shows that species with shade-tolerant seedlings can also recruit as trees early in succession. For these species, early tree colonization enhances seedling and sapling recruitment during the first 20–30 yr of succession, due to local seed rain. Species abundance and size distribution depend strongly on chance colonization events early in succession (Chazdon 2008). Other studies have shown that mature forest species are able to colonize early in succession (Finegan 1996, van Breugel et al. 2007, Franklin &amp; Rey 2007, Ochoa-Gaona et al. 2007), emphasizing the importance of initial floristic composition in the determination of successional pathways and rates of forest regrowth. On the other hand, significant numbers of species in our sites (40% overall and the majority of rare species) colonized only after canopy closure, and these species may not occur as mature individuals until decades after agricultural abandonment.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Classifying functional types based on functional traits with low plasticity, such as wood density and seed size, could potentially serve as robust proxies for demographic variables (Poorter et al. 2008, Zhang et al. 2008).</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>CONDIT, R., S. P. HUBBELL, AND R. B. FOSTER. 1996. Assessing the response of plant functional types in tropical forests to climatic change. J. Veg. Sci. 7: 405–416. DALLING, J. S., AND S. P. HUBBELL. 2002. Seed size, growth rate and gap microsite conditions as determinants of recruitment success for pioneer species. J. Ecol. 90: 557–568. FINEGAN, B. 1996. Pattern and process in neotropical secondary forests: The first 100 years of succession. Trends Ecol. Evol. 11: 119–124. POORTER, L., S. J. WRIGHT, H. PAZ, D. D. ACKERLY, R. CONDIT, G. IBARRA-MANRI´QUEZ, K. E. HARMS, J. C. LICONA, M.MARTI´NEZ-RAMOS, S. J. MAZER, H. C. MULLER-LANDAU, M. PEN˜ A-CLAROS, C. O. WEBB, AND I. J. WRIGHT. 2008. Are functional traits good predictors of demographic rates? Evidence from five Neotropical forests. Ecology 89: 1908–1920. ZHANG, Z. D., R. G. ZANG, AND Y. D. QI. 2008. Spatiotemporal patterns and dynamics of species richness and abundance of woody plant functional groups in a tropical forest landscape of Hainan Island, South China. J. Integr. Plant Biol. 50: 547–558.</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Baraloto et al. 2010. Functional trait variation and sampling strategies in species-rich plant communities</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Therecent growth of large functional trait data bases has been fuelled by standardized protocols forthe measurement of individual functional traits and intensive efforts to compile trait data(Cornelissen etal. 2003; Chave etal. 2009). Nonetheless, there remains no consensusfor the most appropriate sampling design so that traits can be scaled from the individuals on whom measurements are made to the community or ecosystem levels at which infer- ences are drawn (Swenson etal. 2006,2007,Reich,Wright &amp; Lusk 2007;Kraft,Valencia &amp; Ackerly 2008).</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>However, the fast pace of development of plant trait meta-analyses also suggests that trait acquisition in the field is a factor limiting the growth of plant trait data bases.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>We measured traits for every individual tree in nine 1-ha plots in tropical lowland rainforest (N = 4709). Each plant was sampled for 10 functional traits related to wood and leaf morphology and ecophysiology. Here, we contrast the trait means and variances obtained with a full sampling strategy with those of other sampling designs used in the recent literature, which we obtain by simulation. We assess the differences in community- level estimates of functional trait means and variances among design types and sampling intensities. We then contrast the relative costs of these designs and discuss the appropriateness of different sampling designs and intensities for different questions and systems.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Falar que a escolha das categorias de sucessão e dos parâmetros ou característica dos indivíduos que serão utilizadas dependera da facilidade de coleta dos dados e do custo monetário e temporal.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Ver se classifica sucessão por densidade de tronco para citar no artigo como exemplo de outros atributos além de germinação e ver se e custoso no tempo e em dinheiro</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Intensas amostragens de experimentos simples tem maior retorno em acurácia de estimativa e de custo tb.</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>With regard to estimating mean trait values, strategies alternative to BRIDGE were consistently cost-effective. On the other hand, strategies alternative to BRIDGE clearly failed to accurately estimate the variance of trait values. This indicates that in situations where accurate estimation of plotlevel variance is desired, complete censuses are essential.</w:t>
</w:r>
</w:p>
<w:p>
<w:pPr>
<w:pStyle w:val="BodyText"/>
</w:pPr>
<w:r>
<w:t>
                            
                            Isso significa que estudos de característica de história de vida compensam? Ver nos m&amp;m.
                        </w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>We suggest that, in these studies, the investment in complete sampling may be worthwhile for at least some traits.</w:t>
</w:r>
</w:p>
<w:p>
<w:pPr>
<w:pStyle w:val="BodyText"/>
</w:pPr>
<w:r>
<w:t>
                            
                            Falar que isso corrobora nossa sugestão de utilizar poucas medidas, mas que elas sejam confiáveis.
                        </w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sect>
</w:body>
</w:wordDocument>
