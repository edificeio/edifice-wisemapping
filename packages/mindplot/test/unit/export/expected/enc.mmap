<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="1W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="4W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Tolerância a sombra está relacionada com persistência e não com crescimento"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Espécies pioneiras crescem mais rápido do que as não pioneiras"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Poorter 1999. Functional Ecology. 13:396-410"><ap:Font Color="ff000000"/></ap:Text><ap:Color FillColor="ffcccccc"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="1754nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2454nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Falar no artigo que esse trabalho fala que é inadequada a divisão entre pioneira e não pioneira devido a grande variação que há entre elas. Além de terem descoberto que durante a ontogenia a resposta a luminosidade muda dentro de uma mesma espécie. Porém recomendar que essa classificação continue sendo usada em curto prazo enquanto não há informações confiáveis suficiente para esta simples classificação. Outras classificações como esta do artigo são bem vinda, contanto que tenham dados confiáveis. Porém dados estáticos já são difíceis de se obter, dados temporais, como taxa de crescimento em diâmetro ou altura, são mais difíceis ainda. Falar que vários tipos de classificações podem ser utilizadas e quanto mais detalhe melhor, porém os dados é que são mais limitantes. Se focarmos em dados de germinação e crescimento limitantes, como sugerem sainete e whitmore, da uma idéia maismrápida e a curto prazo da classificação destas espécies. Depois com o tempo conseguiremos construir classificações mais detalhadas e com mais dados confiáveis. "><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2254nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2354nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2554nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2654nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2754nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2854nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2954nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3054nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3154nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Chazdon 2010. Biotropica. 42(1): 31–40"><ap:Font Color="ff000000"/></ap:Text><ap:Color FillColor="ffcccccc"/><ap:SubTopicShape/></ap:Topic><ap:Topic OId="5W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="6W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="7W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="8W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="9W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Falar que a escolha das categorias de sucessão e dos parâmetros ou característica dos indivíduos que serão utilizadas dependera da facilidade de coleta dos dados e do custo monetário e temporal."><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Ver se classifica sucessão por densidade de tronco para citar no artigo como exemplo de outros atributos além de germinação e ver se e custoso no tempo e em dinheiro"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1354nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Intensas amostragens de experimentos simples  tem maior retorno em acurácia de estimativa e de custo tb."><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1454nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1554nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Baraloto et al. 2010. Functional trait variation and sampling strategies in species-rich plant communities"><ap:Font/></ap:Text><ap:Color FillColor="ffcccccc"/><ap:SubTopicShape/></ap:Topic></ap:SubTopics><ap:Text PlainText="Artigos GF comentários interessantes"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>