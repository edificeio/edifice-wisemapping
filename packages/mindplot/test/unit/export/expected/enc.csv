Artigos GF comentários interessantes 
,
            Poorter 1999. Functional Ecology. 13:396-410 
,
            ,
            Espécies pioneiras crescem mais rápido do que as não pioneiras 
,
            ,
            ,
            Tolerância a sombra está relacionada com persistência e não com crescimento 
,
            Chazdon 2010. Biotropica. 42(1): 31–40 
,
            ,
            Falar no artigo que esse trabalho fala que é inadequada a divisão entre pioneira e não pioneira devido a grande variação que há entre elas. Além de terem descoberto que durante a ontogenia a resposta a luminosidade muda dentro de uma mesma espécie. Porém recomendar que essa classificação continue sendo usada em curto prazo enquanto não há informações confiáveis suficiente para esta simples classificação. Outras classificações como esta do artigo são bem vinda, contanto que tenham dados confiáveis. Porém dados estáticos já são difíceis de se obter, dados temporais, como taxa de crescimento em diâmetro ou altura, são mais difíceis ainda. Falar que vários tipos de classificações podem ser utilizadas e quanto mais detalhe melhor, porém os dados é que são mais limitantes. Se focarmos em dados de germinação e crescimento limitantes, como sugerem sainete e whitmore, da uma idéia maismrápida e a curto prazo da classificação destas espécies. Depois com o tempo conseguiremos construir classificações mais detalhadas e com mais dados confiáveis.  
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
             
,
            Baraloto et al. 2010. Functional trait variation and sampling strategies in species-rich plant communities 
,
            ,
             
,
            ,
             
,
            ,
             
,
            ,
            Falar que a escolha das categorias de sucessão e dos parâmetros ou característica dos indivíduos que serão utilizadas dependera da facilidade de coleta dos dados e do custo monetário e temporal. 
,
            ,
            Ver se classifica sucessão por densidade de tronco para citar no artigo como exemplo de outros atributos além de germinação e ver se e custoso no tempo e em dinheiro 
,
            ,
            Intensas amostragens de experimentos simples  tem maior retorno em acurácia de estimativa e de custo tb. 
,
            ,
             
,
            ,
             
